pgmpy/tests/test_readwrite/test_ProbModelXML.py

```diff
@@ -85,6 +93,7 @@ def setUp(self):
         self.reader_file = ProbModelXMLReader(path=StringIO(string))
 
     def test_comment(self):
+        print(self.reader_string.probnet['DecisionCriteria'])
         comment_expected = "Student example model from Probabilistic Graphical Models: Principles and Techniques by Daphne Koller"
```

Comment:
Make sure that ```comment_expected``` is under the character limit of 120. If it exceeds, you can replace it as

```python
comment_expected = ("Student example model from Probabilistic "
                    "Graphical Models: Principles and Techniques by Daphne Koller")
```